src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
Consider using 'minmax(200px, 1fr)' instead of a fixed size for grid-template-columns to make the layout more flexible. This allows the columns to grow if there is extra space, while still respecting the minimum size of 200px.